pyuvdata/uvdata/uvdata.py

```diff
@@ -9595,6 +9641,7 @@ def resample_in_time(
 
     def frequency_average(
         self,
+        *,
         n_chan_to_avg,
```

Comment:
along with the other resampling methods, should `n_chan_to_avg` be a positional argument?

Reply:
probably.